modules/sandboxed-containers-selecting-nodes.adoc

```diff
@@ -3,5 +3,31 @@
 // * sandboxed_containers/deploying_sandboxed_containers.adoc
 
 [id="sandboxed-containers-selecting-nodes_{context}"]
+= Selecting nodes for {sandboxed-containers-first}
 
-= Selecting nodes for OpenShift sandboxed containers
+You can selectively install the Kata runtime on specific workers.
+
+.Prerequisites
+
+* You have {product-title} 4.8 installed on your cluster.
+* You have installed the OpenShift CLI (oc).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Identify the labels you want to use for selecting your nodes. For this example, select nodes with labels to be chosen as candidates to run on your {sandboxed-containers-first} workloads. If the nodes exist, they are selected.
+.. To apply a label to a node, run the following command: `oc label node <worker_node_name> custom-kata=test`. This labels your worker node with the `custom-kata` label that has a value of `test`.
```

Comment:
custom-kata=test is an example that comes from the earliest days of developing this feature but the name has no meaning. We can safely change this to something more meaningful like "custom-kata-machine-pool=true". 

We can also add a sentence that the label-name  can be chosen freely. "If you wish, you can change the label "custom-kata-machine-pool=true" to something of your choice.

Reply:
For clarity, you could even avoid having `kata` in the label at all. What about `oc label node <worker_node_name> <label>=<value>` ? And then stating that this labels your worker node with the `<label>` label that has a value of `<value>`.